Makefile.am

```diff
@@ -30,7 +30,54 @@ LDADD = \
 AM_DISTCHECK_CONFIGURE_FLAGS = --with-bashcompdir='$$(datarootdir)/bash-completion/completions'
 
 # keep me sorted
-bin_PROGRAMS = \
+bin_PROGRAMS =
+FAPI_CFLAGS = $(EXTRA_CFLAGS) $(TSS2_FAPI_CFLAGS) $(CODE_COVERAGE_CFLAGS)
+TESTS =
+
+if HAVE_FAPI
+
+bin_PROGRAMS += \
+    tools/fapi/tss2_decrypt \
+    tools/fapi/tss2_encrypt \
+    tools/fapi/tss2_list \
+    tools/fapi/tss2_changeauth \
+    tools/fapi/tss2_delete \
+    tools/fapi/tss2_import \
+    tools/fapi/tss2_getinfo \
+    tools/fapi/tss2_createkey \
+    tools/fapi/tss2_createseal \
+    tools/fapi/tss2_exportkey \
+    tools/fapi/tss2_getcertificate \
+    tools/fapi/tss2_getplatformcertificates \
+    tools/fapi/tss2_gettpmblobs \
+    tools/fapi/tss2_getappdata \
+    tools/fapi/tss2_setappdata \
+    tools/fapi/tss2_setcertificate \
+    tools/fapi/tss2_sign \
+    tools/fapi/tss2_verifysignature \
+    tools/fapi/tss2_verifyquote \
+    tools/fapi/tss2_createnv \
+    tools/fapi/tss2_nvextend \
+    tools/fapi/tss2_nvincrement \
+    tools/fapi/tss2_nvread \
+    tools/fapi/tss2_nvsetbits \
+    tools/fapi/tss2_nvwrite \
+    tools/fapi/tss2_getdescription \
+    tools/fapi/tss2_setdescription \
+    tools/fapi/tss2_pcrextend \
+    tools/fapi/tss2_quote \
+    tools/fapi/tss2_pcrread \
+    tools/fapi/tss2_authorizepolicy \
+    tools/fapi/tss2_exportpolicy \
+    tools/fapi/tss2_import \
+    tools/fapi/tss2_provision \
+    tools/fapi/tss2_getrandom \
+    tools/fapi/tss2_unseal \
+    tools/fapi/tss2_writeauthorizenv
```

Comment:
+1 for placing these into a subdir, read my mind.

Reply:
awesome, marked as resolved... ;-)